core/src/main/java/org/springframework/security/jackson2/SecurityJackson2Modules.java

```diff
@@ -225,7 +225,7 @@ public JavaType typeFromId(DatabindContext context, String id) throws IOExceptio
 			JavaType result = delegate.typeFromId(context, id);
```

Comment:
Please update the Apache license header in this file. For more details see the [contribution guidelines](https://github.com/spring-projects/spring-security/blob/master/CONTRIBUTING.md#update-apache-license-header-to-modified-files-as-necessary).